sonar-db/src/test/java/org/sonar/db/permission/GroupPermissionDaoTest.java

```diff
@@ -279,4 +280,36 @@ public void select_group_permissions_by_group_names_on_project_permissions() {
     assertThat(underTest.selectGroupPermissionsByGroupNamesAndProject(dbSession, Collections.emptyList(), project.getId())).isEmpty();
   }
 
+  @Test
+  public void selectGroupPermissions() {
+    permissionDb.addGlobalPermissionToGroup("perm1", /* anyone */null);
```

Comment:
you could use a constant called `GROUP_ANYONE` which value is `null` rather than rely on a comment